qa/tasks/qemu.py

```diff
@@ -324,8 +325,9 @@ def _teardown_nfs_mount(remote, client, service_name):
         'sudo', 'rm', '-r', '/export'
     ])
     log.info("Deleting export from /etc/exports...")
+    delete_mount_point = '/' + client + '/d'
```

Comment:
same comment here

Reply:
On teardown, we should make sure that each qemu just unmount its mount-point. If using /^\/export\//d, the first qemu will unmount all mount points which other qemu maybe still is using these mount point.

Reply:
Understood -- just asking that it re-use the existing `export_dir` variable.

Reply:
done.